src/components/App.tsx

```diff
@@ -194,15 +196,27 @@ const App = () => {
               </RequireScope>
             }
           />
+
           <Route path="schedulecourses" element={<Scheduler />} />
           {/* Static Content Pages */}
+
           <Route path="about" element={<About />} />
           <Route path="faq" element={<FAQ />} />
           <Route path="team" element={<Team />} />
+          <Route
+            path="admin"
+            element={
+              <RequireScope token={apiToken} name="admin.dashboard">
```

Comment:
This route name should be just `admin` (since the path name is also just `/admin`)